README.md

```diff
@@ -1,13 +1,14 @@
 # Foreman Pulp Plugin
 
-Foreman project plugin for Pulp allowing Katello hosts to interact with Pulp application services for content management
-While this plugin is part of the Foreman project, it can only be used with Katello as Foreman is not content aware without the Katello plugin.
+Foreman project plugin for Pulp allowing Katello hosts to interact with Pulp application services for content management. While this plugin is part of the Foreman project, it can only be used with Katello as Foreman is not content aware without the Katello plugin.
 
 ## Getting Started
 
 The Foreman project provides documentation on this specific plugin installation from multiple methods, distribution specific RPM, distribution specific DEB and direct manual build from source.
```

Comment:
In this particular case we don't build a DEB. I think that could have been copy pasted from a template.